serverhttpy/models/path.py

```diff
@@ -10,5 +16,15 @@ def __eq__(self, other):
     def __hash__(self):
         return hash(self.path)
 
-    def __init__(self, path: str):
-        self.path = path
+    def parse(self,path: str):
```

Comment:
Code style: missing space after comma; please apply in other places if any other exist